tests/helper/helper_kubectl.go

```diff
@@ -235,3 +237,25 @@ func (kubectl KubectlRunner) VerifyResourceToBeDeleted(ri ResourceInfo) {
 	}
 	Expect(deletedOrMarkedToDelete()).To(BeTrue())
 }
+
+// GetAnnotationsDeployment gets the annotations from the deployment
+// belonging to the given component, app and project
+func (kubectl KubectlRunner) GetAnnotationsDeployment(componentName, appName, projectName string) map[string]string {
+	var mapOutput = make(map[string]string)
+
+	selector := fmt.Sprintf("--selector=%s=%s,%s=%s", labels.ComponentLabel, componentName, applabels.ApplicationLabel, appName)
+	output := CmdShouldPass(kubectl.path, "get", "deployment", selector, "--namespace", projectName,
```

Comment:
It seems the difference between this function and the oc counter part is just the `oc.path` and `kubectl.path` https://github.com/openshift/odo/pull/4766/files#diff-b70710c3c2ee60424c00f89c50f99e6446009b2452baf5e6eaf3d2dc187aec12R694

Can we extract this into a separate function and make it common across both?

Reply:
Fixed